docs/tutorial/part06.rst

```diff
@@ -2,5 +2,648 @@
+
+On your ``fetch_api/utils.py`` the ``fetch_node_details`` function change put this above the return
+statement::
+
+        # Make sure to return an empty array if not connections
+        node_details['node_connections'] = []
+        if (hasattr(node, 'serialize_connections')):
+            node_details['node_connections'] = node.serialize_connections
+
+So the function should be looking something like this::
+
+    def fetch_node_details(node_info):
+        node_type       = node_info['node_type']
+        node_id         = node_info['node_id']
+        node            = MODEL_ENTITIES[node_type].nodes.get(node_id=node_id)
+        node_details    = node.serialize
+
+        # Make sure to return an empty array if not connections
+        node_details['node_connections'] = []
+        if (hasattr(node, 'serialize_connections')):
+            node_details['node_connections'] = node.serialize_connections
+
+        return node_details
+
+Return the json to the frontend
+===============================
+
+Now if you call your functions ``fetch_node_details`` and ``fetch_nodes`` should be returning the
+same data but in a way that is JSON parsable, so let's change a couple things in order of returning
+this data that we need.
+
+In your project settings file for us it is ``paradise_papers_search/settings/base.py`` but if you
+are using a costume project it may be different for you, add the following::
+
+    # Rest-Framework settings
+    REST_FRAMEWORK = {
+        'DEFAULT_RENDERER_CLASSES': (
+            'rest_framework.renderers.JSONRenderer',
+        )
+    }
+
+This is going to cause that when we invoke the render method on our APIs views the method will call
+the JSON renderer rather than just the HTTP one.
+
+So once again we will clean our ``fetch_api/views.py`` file and leave it like this::
+
+    from rest_framework.views import APIView
+    from rest_framework.response import Response
+    from .models.helpers import fetch_nodes
+
+    class GetNodesData(APIView):
+        def get(self, request):
+            fetch_info = {
+                'node_type': request.GET.get('t', 'Entity'),
+                'name': request.GET.get('q', ''),
+                'country': request.GET.get('c', ''),
+                'jurisdiction': request.GET.get('j', ''),
+                'sourceID': request.GET.get('s', ''),
+                'limit': 10,
+                'skip': int(request.GET.get('p', 1)),
+            }
+            nodes = fetch_nodes(fetch_info)
+            data = {
+                'response': {
+                    'status': '200',
+                    'rows': len(nodes),
+                    'data': nodes,
+                },
+            }
+            return Response(data)
+
+Here we are just taking the query parameters off the request and parsing them to pass them to the
+respective fetch function.
+
+This lines here::
+
+                'limit': 10,
+                'skip': int(request.GET.get('p', 1)),

```

Comment:
I think it is better if we change the name ``skip`` for ``page``.
What do you think?